components/about/About.module.css

```
@@ -43,6 +43,103 @@
+  padding-bottom: 0;
+  border-bottom: 0;
+}
+
+.gdrcContent h2,
+.gdrcContent h3,
+.gdrcContent h4,
+.gdrcContent h5 {
+  float: none;
+  margin-top: 0;
+  color: #efeff1;
+  font-family: "Montserrat", sans-serif !important;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)